src/genie/libs/parser/iosxe/show_redundancy.py

```diff
@@ -0,0 +1,94 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# =======================================
+# Schema for:
+#  * 'show redundancy switchover history'
+# =======================================
+class ShowRedundancySwitchoverHistorySchema(MetaParser):
+    """Schema for show redundancy switchover history."""
+
+    schema = {
+        Optional("index"): {
+            Any(): {
+                "current_active": int,
+                "previous_active": int,
+                "switchover_reason": str,
+                "switchover_time": str,
+            },
+        }
+    }
+
+
+# =======================================
+# Parser for:
+#  * 'show redundancy switchover history'
+# =======================================
+class ShowRedundancySwitchoverHistory(ShowRedundancySwitchoverHistorySchema):
+    """Parser for show redundancy switchover history"""
+
+    cli_command = ['show redundancy switchover history']
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command[0])
+
+        # Index  Previous  Current  Switchover             Switchover
+        #        active    active   reason                 time
+        # -----  --------  -------  ----------             ----------
+        #    1       1        2     Active lost GW         03:26:09 UTC Wed Sep 23 2020
+        #    2       2        1     user forced            05:40:13 UTC Thu Oct 1 2020
+
+
+        #    1       1        2     Active lost GW         03:26:09 UTC Wed Sep 23 2020
+        switchover_history_capture = re.compile(
+            # 1
+            r"^(?P<index>\d+)\s+"
+            # 1
+            r"(?P<previous_active>\d+)\s+"
+            # 2
+            r"(?P<current_active>\d+)\s+"
+            # Active lost GW
+            r"(?P<switchover_reason>.+)\s+"
+            # 03:26:09 UTC Wed Sep 23 2020
+            r"(?P<switchover_time>\d+:\d+:\d+\sUTC\s\S+\s\S+\s\d+\s\d+)$"
```

Comment:
I see this `15:35:42 edt Wed`, as possible example `http://bowe.id.au/michael/isp/DOCSIS/collected-references/BRKSPG-2501-2018.pdf` and don't think utc can be presumed.

Reply:
Yeah I was wondering if they can set the timezone. I'll drop in `\S` instead!